data/zittogustavo.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Gustavo Zitto",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Developer",
+  "avatar": "https://github.com/zitogustavo.png",
+  "tags": ["javascript", "typescript", "HTML", "PHP", "React"],
+  "socials": [
+    { "platform": "github", "url": "https://github.com/zittogustavo" }
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/zittogustavo",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```